### PR TITLE
OpenAPI support for native-image

### LINKED ISSUE #1291: OpenAPI in MP native-image
Related to #716 

Support Microprofile OpenAPI in `native-image`

Depends on #1288

### PR DESCRIPTION
**Pull request by tomas-langer**
Resolves #1291 

Updated integration test to include OpenAPI with annotated JAX-RS resource method.

### CODE REVIEW

File: tests/integration/native-image/mp-1/pom.xml [hunk 1]
```diff
@@ -100,26 +55,20 @@
...
-            <groupId>org.glassfish.jersey.media</groupId>
-            <artifactId>jersey-media-json-binding</artifactId>
+            <groupId>io.helidon.microprofile</groupId>
+            <artifactId>helidon-microprofile-access-log</artifactId>
             <scope>runtime</scope>
         </dependency>
         <dependency>
             <groupId>org.jboss</groupId>
             <artifactId>jandex</artifactId>
             <version>2.1.1.Final</version>
```
Comment (tjquinno) [inline tests/integration/native-image/mp-1/pom.xml]: I know this was not changed as part of this PR but jandex is in `<dependencyManagement>` in `dependencies/pom.xml` so the version should not be needed here.

Comment (tomas-langer) [reply tests/integration/native-image/mp-1/pom.xml]: Will fix in other tests as well.

Comment (tomas-langer) [reply tests/integration/native-image/mp-1/pom.xml]: Fixed.

**Comment by tomas-langer** [comment]
@barchetta - we have dependencies in `mp-2` integration tests that are required for it to work - both module dependencies and maven plugins...

**Pull request merged**